api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java

```diff
@@ -58,4 +58,10 @@
     @JsonProperty("address")
     @ApiModelProperty(notes = "The address of the claimant")
     private AddressDTO address;
+
+    @NotNull
+    @Pattern(regexp = "^\\+44\\d{9,10}$", message = "invalid uk phone number, must be in +44 format, e.g. +447123456789")
+    @JsonProperty("phoneNumber")
+    @ApiModelProperty(notes = "The claimant's phone number. May be used for verification purposes.")
```

Comment:
Its probably worthwhile mentioning that is must be in UK format here.

Reply:
Done.